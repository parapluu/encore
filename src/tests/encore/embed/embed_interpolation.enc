class Main {
  def main() : void {
    this.foo(2)
  }

  def foo(y: int): void {
    let x = 1 in {
      embed void printf("x=%llu\n", #{x}); end;
      embed void printf("y=%llu\n", #{y}); end;
      print "(btw: helloponyworld)"
    }
  }
}
