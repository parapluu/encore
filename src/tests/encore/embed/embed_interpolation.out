x=1
y=2
(btw: helloponyworld)
